web/satellite/src/views/Dashboard.vue

```diff
@@ -41,94 +39,93 @@
     import { PROJECT_USAGE_ACTIONS } from '@/store/modules/usage';
 
     @Component({
-    mounted: async function() {
-        setTimeout(async () => {
-            try {
-                await this.$store.dispatch(USER_ACTIONS.GET);
-            } catch (error) {
-                await this.$store.dispatch(APP_STATE_ACTIONS.CHANGE_STATE, AppState.ERROR);
-                await this.$store.dispatch(NOTIFICATION_ACTIONS.ERROR, error.message);
-                await this.$router.push(RouteConfig.Login.path);
-                AuthToken.remove();
-
-                return;
-            }
+        components: {
+            NavigationArea,
+            DashboardHeader,
+        }
+    })
+    export default class Dashboard extends Vue {
+        public async mounted(): Promise<void> {
+            setTimeout(async () => {
+                // TODO: combine all project related requests in one
+                try {
+                    await this.$store.dispatch(USER_ACTIONS.GET);
+                } catch (error) {
+                    await this.$store.dispatch(APP_STATE_ACTIONS.CHANGE_STATE, AppState.ERROR);
+                    await this.$store.dispatch(NOTIFICATION_ACTIONS.ERROR, error.message);
+                    await this.$router.push(RouteConfig.Login.path);
+                    AuthToken.remove();
 
-            let projects: Project[] = [];
+                    return;
+                }
 
-            try {
-                projects = await this.$store.dispatch(PROJECTS_ACTIONS.FETCH);
-            } catch (error) {
-                await this.$store.dispatch(NOTIFICATION_ACTIONS.ERROR, error.message);
+                let projects: Project[] = [];
 
-                return;
-            }
+                try {
+                    projects = await this.$store.dispatch(PROJECTS_ACTIONS.FETCH);
+                } catch (error) {
+                    await this.$store.dispatch(NOTIFICATION_ACTIONS.ERROR, error.message);
```

Comment:
Should we add more general error description before displaying `error.message`?
something like ``Unable to fetch projects. ${error.message}``

Reply:
yeah